lib/install/rust.js

```diff
@@ -139,7 +139,7 @@ module.exports.toolchainPath = () => {
     var values = fs.readdirSync(sdkPlatformPath);
 
     for (var i = 0; i < values.length; i++) {
-      if (values[i].match(/^toolchain\-/)) {
+      if (values[i].match(/^toolchain\-mipsel/)) {
```

Comment:
Clarifying q: this is just checking that the string in `values[i]` includes `"toolchain-mipsel/"`?